module/zfs/zstd/freebsd/stddef.h

```diff
@@ -0,0 +1,3 @@
+/* This file is in the public domain */
+/* $FreeBSD$ */
+#include "zstd_kfreebsd.h"
```

Comment:
I wouldn't expect us to need any of these freebsd wrappers since `zstd` builds fine on Linux.  Can you look in to removing them, no point keep compatibility code on top of the compatibility code.